exonum/build.rs

```diff
@@ -35,17 +50,28 @@ fn main() {
     file.write_all(user_agent.as_bytes())
         .expect("Unable to data to file");
 
-    protoc_rust::run(protoc_rust::Args {
-        out_dir: "src/encoding/protobuf",
-        input: &get_proto_files("src/encoding/protobuf/proto/")
-            .iter()
-            .map(|s| s.as_ref())
-            .collect::<Vec<_>>(),
-        includes: &["src/encoding/protobuf/proto"],
-        customize: Customize {
-            ..Default::default()
-        },
-    }).expect("protoc");
+    protoc_generate(
+        "src/encoding/protobuf",
+        "src/encoding/protobuf/proto/",
+        &["src/encoding/protobuf/proto"],
+    );
+
+    // Exonum external tests.
+    protoc_generate(
```

Comment:
It will not work without creating separate crate for tests or something like that, and I think it is not an issue.
https://github.com/rust-lang/cargo/issues/4789